.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,74 @@
+name: publish distributions
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - v*
+  pull_request:
+    branches:
+    - main
+  workflow_dispatch:
+    inputs:
+      publish:
+        type: choice
+        description: 'Publish to TestPyPI?'
+        options:
+        - false
+        - true
```

Comment:
@peterridolfi for context, once the tokens are added to the GitHub secrets if you got to https://github.com/peterridolfi/Pyhf-to-Combine-converter/actions/workflows/publish-package.yml you can publish to TestPyPI by selecting the "Run workflow" button and then toggling from `false` to `true.

![publish_to_testpypi](https://user-images.githubusercontent.com/5142394/185255477-edf2878b-99e1-47d8-8bec-2e6636673745.png)